hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoTransferSuite.java

```diff
@@ -285,7 +283,6 @@ private HapiSpec okToRepeatSerialNumbersInWipeList() {
     private HapiSpec okToRepeatSerialNumbersInBurnList() {
         return defaultHapiSpec("okToRepeatSerialNumbersInBurnList")
                 .given(
-                        snapshotMode(FUZZY_MATCH_AGAINST_HAPI_TEST_STREAMS),
```

Comment:
We _don't_ want this anymore?

Reply:
It was failing in my local for that reason. Not sure why. I will start with `CryptoTransferSuite` Fuzzy matching and will enable it.